com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -706,29 +736,59 @@ private void ApplyAuthoritativeState()
                     interpolatedRotAngles.z = isTeleporting || !Interpolate ? networkState.RotAngleZ : eulerAngles.z;
                 }
             }
+            else if (Interpolate && SynchronizeRotation)
+            {
+                var eulerAngles = m_RotationInterpolator.GetInterpolatedValue().eulerAngles;
+                interpolatedRotAngles = eulerAngles;
+                if (SyncRotAngleY)
+                {
+                    interpolatedRotAngles.x = m_RotationInterpolator.GetInterpolatedValue().eulerAngles.x;
```

Comment:
I'm a bit confused here... it looks to me, following up to where interpolatedRotAngles comes from, like you're doing, effectively, `interpolatedRotAngles.x = interpolatedRotAngles.x`. Am I missing something here?

Simplifying what I see:
```C#
interpolatedRotAngles = m_RotationInterpolator.GetInterpolatedValue().eulerAngles;
interpolatedRotAngles.x = m_RotationInterpolator.GetInterpolatedValue().eulerAngles.x;
```

Reply:
🤯

Reply:
That is a copy and paste bug and I am fixing it post-haste.